Podcast/PlayerEpisodeDetailView.swift

```diff
@@ -44,7 +44,7 @@ class PlayerEpisodeDetailView: UIView, UIGestureRecognizerDelegate {
     let episodeTitleShowMoreSpacing: CGFloat = 12
     let dateLabelShowMoreTopOffset: CGFloat = 5.5
     let seeMoreButtonWidth: CGFloat = 100
-    let seeMoreButtonHeight: CGFloat = 10
+    let seeMoreButtonHeight: CGFloat = 20
```

Comment:
I don't think this fixes the "Show More" issue
<img width="256" alt="screen shot 2018-03-09 at 2 32 30 am" src="https://user-images.githubusercontent.com/14966713/37195798-42b07796-2342-11e8-85e1-12c9cef605ee.png">

Reply:
should be fixed now